hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicSecurityConfig.java

```diff
@@ -121,22 +121,27 @@ public SecurityConfig setClientPolicyConfig(PermissionPolicyConfig policyConfig)
      */
     @Override
     public Set<PermissionConfig> getClientPermissionConfigs() {
-        return Collections.unmodifiableSet(securityService.getClientPermissionConfigs());
+        Set<PermissionConfig> permissionConfigs = securityService != null
+                ? securityService.getClientPermissionConfigs()
+                : staticSecurityConfig.getClientPermissionConfigs();
+        return Collections.unmodifiableSet(permissionConfigs);
     }
 
     /**
      *  Updates client permission configuration cluster-wide.
      */
     @Override
     public SecurityConfig setClientPermissionConfigs(Set<PermissionConfig> permissions) {
+        if (securityService == null) {
```

Comment:
It's `UnsupportedOperationException`, because `securityService == null` means either OSS code is used or security is not enabled. 
In other parts of dynamic configuration work, `UnsupportedOperationException` is thrown for similar cases.